ecl/eclagent/eclagent.cpp

```diff
@@ -3546,10 +3492,10 @@ extern int HTHOR_API eclagent_main(int argc, const char *argv[], StringBuffer *
                 IExtendedWUInterface * extendedWu = queryExtendedWU(daliWu);
                 extendedWu->copyWorkUnit(standAloneWorkUnit, true, true);
                 wuid.set(daliWu->queryWuid());
-                globals->setProp("WUID", wuid.str());
+                cmdLineArgs->setProp("WUID", wuid.str());  // So it can be retrieved by getenv
```

Comment:
Should this be lowercase "wuid"?

Reply:
If there was any ECL code calling getenv to get it, it would previously have been there as WUID, so I left it.

getenv should probably be made case-insensitive, but that's a different (existing) bug.